src/Form/OptionsApi.php

```diff
@@ -86,10 +86,12 @@ public function fetch()
      * @param array $data
      * @return string
      */
-    protected function field(string $field, array $data)
+    protected function field(string $field, array $data): string
     {
         $value = $this->$field();
-        return Str::template($value, $data);
+        return Str::template($value, $data, null, '{{', '}}', function ($result) {
+            return strip_tags($result);
```

Comment:
`strip_tags()` does not protect against XSS attacks. Also it would remove tag-like text completely. For backwards compatibility and security it would be better if the API return values were escaped with `Toolkit\Escape::html()` (that's what `v-text` does in Vue as well).